components/common/Text.tsx

```diff
@@ -8,9 +8,13 @@ interface TextProps {
   onPress?: () => void;
   centered?: boolean;
   marginHorizontal?: number;
+  isAnimated?: boolean;
+  animationTrigger?: any;
+  animationSequence?: () => void;
+  scaleAnimation?: any;
 }
 
-const LText = ({ children, style, centered, marginHorizontal = 0, size, onPress }: TextProps) => {
+const LText = ({ children, style, centered, marginHorizontal = 0, size, onPress, isAnimated = false, animationTrigger, animationSequence, scaleAnimation }: TextProps) => {
```

Comment:
making all of these optional may cause errors during usage instead, we could have added the following:
```js
animation:? {
  isAnimated: boolean;
  animationTrigger: any;
  animationSequence: () => void;
  scaleAnimation: any;
}
```